projects/forge-lib/src/lib/shared/form-editor-components/concrete/date-time-editor/display/date-time-editor-display.component.html

```diff
@@ -0,0 +1,15 @@
+<div sws-form input-active full-width>
+  <input [(ngModel)]="this.component.display.placeholder" sws-form__input type="'text'" [name]="'placeholder_input'"
+      onkeyup="this.setAttribute('value', this.value);">
+  <label sws-form__label [for]="'label_input'">Placeholder</label>
```

Comment:
language file

Reply:
language file will be added on later phases